tests/test_daemon_extended.py

```
@@ -0,0 +1,176 @@
+import time
+import urllib.request
+from pathlib import Path
+
+import pytest
+
+from gradata.daemon import GradataDaemon
+
+
+# ── Fixture: spin up a daemon on a random port ─────────────────────────
+
+@pytest.fixture()
```

Ruff (0.15.9)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)